Code/Framework/AzCore/AzCore/Math/VectorN.inl

```diff
@@ -233,19 +233,19 @@ namespace AZ
         return returnValue;
     }
 
-    AZ_MATH_INLINE float VectorN::GetLengthSq() const
+    AZ_MATH_INLINE float VectorN::L1Norm() const
     {
-        float lengthSquared = 0.0f;
+        AZ::Vector4 partialLengths = AZ::Vector4::CreateZero();
         for (AZStd::size_t i = 0; i < m_values.size(); ++i)
         {
-            lengthSquared += m_values[i].GetLengthSq();
+            partialLengths += m_values[i].GetAbs();
```

Comment:
Are the unused elements of the final Vector4 element is 0 when this function is run?

Reply:
Yes, we always ensure unused elements are zero.